.swift-format

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
I think this file is slightly different than the one in `swift-certificates`. Can we align on one please.

Reply:
Aligning on formatting rules will be a _significant_ amount of churn (52,000 lines) that I don't think is worth doing. The aim of this PR is to _reduce_ churn by switching to a tool which produces more stable formatting; in that light the rules chosen follow the existing ones where possible to minimise the changes.

Reply:
I see. On the other hand we are planning to do some significant work in gRPC and that might be a good opportunity to do this one time churn. Having an aligned formatting file across our repos is IMO worth it in the long run. We could take this churn and just tell git to ignore this one commit in blames so it shouldn't even show up.

Reply:
I'm not convinced: we still need to work on and maintain v1 for quite some time so will still bear the cost of the significant churn. Formatters let users avoid having to do the manual work of sticking to style guidelines so the benefit of having a consistent style across projects is very low, IMO.

Reply:
Sure we have to maintain the v1 but this is a one time churn right? If we decide that `swift-format` is the future for our project then IMO we should take that one time churn once and then have everything aligned.

Reply:
Sure, it's one time but it's a _very_ large change. As I said above, I think the benefit of having consistent style across projects is low, especially as we have tools which enforce the style for you.

What we're effectively debating is which style to use. I would much prefer we stick to the style used in this PR because it minimises the changes made, so much so that they can be reviewed with relative ease.